contracts/Portfolio.sol

```diff
@@ -541,19 +482,14 @@ abstract contract PortfolioVirtual is Objective {
 
         // -=- Apply Fee Saving Method -=- //
         {
-            // Fees are saved by incrementing the fee growth accumulator.
-            bool saved = _feeSavingEffects(args.poolId, iteration);
-
             // If the fees are not saved,
             // apply the full next independent amount with fee amount included.
             // Fees were not saved in the claimable balances,
             // so this will re-invest the fees into the pool.
-            if (!saved) {
-                if (_state.sell) {
-                    iteration.virtualX = nextIndependent;
-                } else {
-                    iteration.virtualY = nextIndependent;
-                }
+            if (_state.sell) {
+                iteration.virtualX = nextIndependent;
+            } else {
+                iteration.virtualY = nextIndependent;
```

Comment:
fees are always reinvested now which will always increase the invariant, even above 0.